website/content/en/docs/Config/Mount/_index.md

```diff
@@ -165,3 +165,32 @@ mountType: "wsl2"
 #### Caveats
 - WSL2 file permissions may not work exactly as expected when accessing files that are natively on the Windows disk ([more info](https://github.com/MicrosoftDocs/WSL/blob/mattw-wsl2-explainer/WSL/file-permissions.md))
 - WSL2's disk sharing system uses a 9P protocol server, making the performance similar to [Lima's 9p](#9p) mode ([more info](https://github.com/MicrosoftDocs/WSL/blob/mattw-wsl2-explainer/WSL/wsl2-architecture.md#wsl-2-architectural-flow))
+
+## Mount Inotify
+> **Warning**
+> "mountInotify" is experimental
+
+| ⚡ Requirement | Lima >= 0.19 |
+| ----------------- |--------------------------------------------------------|
+
+The `mountInotify` support enables inotify support for all different mountTypes like 9p, virtiofs etc.
+
+When mountInotify is enabled, hostagent will listen and send inotify events from host machine to guest. 
+
+This support will be enabled only for writable mounts
```

Comment:
I was briefly wondering why we don't set this up for read-only mounts, as the notification direction is from the host to the guest.

But then I realized that this is not a choice, but a limitation, because we need to modify the mounted directories/files inside the guest to re-create the inotify events, so we need write access for that.

So I think it would be useful to explain this here, so people don't start asking to support it on read-only mounts as well.

Reply:
Good catch. Will update it as well

Reply:
I missed this detail, as well.

Mostly because it was not documented in lima.yaml, but only in the documentation (and who reads that).